### PR TITLE
pierwszy commit z komentarzem

### PR DESCRIPTION
**Pull request by mwasalski**

### CODE REVIEW

File: Projects/memoization 2.ipynb [hunk 1]
```diff
@@ -0,0 +1,186 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
```
Comment (marekczuma) [inline Projects/memoization 2.ipynb:4]: Remove it

File: Projects/memoization 2.ipynb [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+     "output_type": "error",
+     "traceback": [
+      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
+      "\u001b[0;31mRecursionError\u001b[0m                            Traceback (most recent call last)",
+      "Cell \u001b[0;32mIn[14], line 7\u001b[0m\n\u001b[1;32m      4\u001b[0m         \u001b[38;5;28;01mreturn\u001b[39;00m n\n\u001b[1;32m      5\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m fibonacci(n\u001b[38;5;241m-\u001b[39m\u001b[38;5;241m1\u001b[39m) \u001b[38;5;241m+\u001b[39m fibonacci(n\u001b[38;5;241m-\u001b[39m\u001b[38;5;241m2\u001b[39m) \n\u001b[0;32m----> 7\u001b[0m f \u001b[38;5;241m=\u001b[39m \u001b[43mfibonacci\u001b[49m\u001b[43m(\u001b[49m\u001b[38;5;241;43m2000\u001b[39;49m\u001b[43m)\u001b[49m\n\u001b[1;32m      8\u001b[0m \u001b[38;5;28mprint\u001b[39m(f)\n\u001b[1;32m      9\u001b[0m \u001b[38;5;28mprint\u001b[39m(\u001b[38;5;124mf\u001b[39m\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mit has take \u001b[39m\u001b[38;5;132;01m{\u001b[39;00mend\u001b[38;5;241m-\u001b[39mstart\u001b[38;5;132;01m}\u001b[39;00m\u001b[38;5;124m seconds\u001b[39m\u001b[38;5;124m\"\u001b[39m)\n",
```
Comment (marekczuma) [inline Projects/memoization 2.ipynb:71]: too long

**Review by marekczuma — APPROVED** [review]
merge it

**Comment by mwasalski** [comment]
approved

**Pull request closed without merging**